plat/arm/board/juno/include/plat_macros.S

```diff
@@ -33,10 +33,16 @@
 #include <cci_macros.S>
 #include <css_macros.S>
 
-/*
- * Required platform porting macros
- * (Provided by included headers)
- */
-
+	/* ---------------------------------------------
+	 * The below required platform porting macro
+	 * prints out relevant platform registers
+	 * whenever an unhandled exception is taken in
+	 * BL31.
+	 * ---------------------------------------------
+	 */
+	.macro plat_crash_print_regs
+	plat_print_interconnect_regs
+	plat_print_gic_regs
```

Comment:
`plat_print_gic_regs()` in css_macros.S should be renamed to `css_print_gic_regs`.